sections/Heros/VideoHero.tsx

```diff
@@ -0,0 +1,240 @@
+'use client';
```

Comment:
So much for their dream of SEO... I can't use the breakpoints without this `use client` and when it's served from the client google can't scrape for SEO. I need to think how to refactor this

Reply:
@charcarr

Reply:
Yes, she makes literally no sense. IMO only titles and important text should be crawlable by SEO. 
If she wants something interactive it just ain't possible.